### DISCUSSION TITLE
Coder upgrade from 0.13 to 0.20

### DISCUSSION THREAD

**Discussion by vl00920760** [coder/coder#6748] [Announcements]
In Coder v2 can I upgrade from 0.13.1 to 0.20.1 ?

I mean directly jumping on the latest version?

I don't see any recommendation in the document.

**Comment by matifali** [comment]
Yes, you can update but be careful as Coder does not support rollbacks.

**Reply by vl00920760** [reply]
Hi, To update on this, I upgraded the mentioned version without any issue and impact. 
We can do using helm upgrade, it does not impact Workspaces, Templates or other configs.

One thing I suggest - Improve the Coder V2 documentation. Your V1 document is better than V2, that give more clarity and brief.

**Reply by matifali** [reply]
Do you have any suggestions on which sections of the docs we need to improve?
